tensorbay/cli/log.py

```diff
@@ -24,32 +27,105 @@
 """
 
 
-def _implement_log(
+def _get_newest_commit(commits: List[Commit]) -> Tuple[Commit, int]:
+    max_timestamp, max_index = 0, 0
+    if len(commits) == 1:
+        return commits[0], 0
+    for index, commit in enumerate(commits):
+        date = commit.committer.date
+        if date > max_timestamp:
+            max_index = index
+            max_timestamp = date
+    return commits[max_index], max_index
```

Comment:
Use `sort` or other sorting methods to keep the `commits` in order?

Reply:
Yes madam!